codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/RestBuilderGeneratorBase.java

```diff
@@ -433,16 +433,26 @@ protected HeaderListMemberSerializer(String inputGetter, String dataSetter, Memb
 
         @Override
         protected Void getDefault(Shape shape) {
```

Comment:
This change should be affecting rest json too right? Are there any cases of that?

Reply:
I'm going to assume that you are referring to protocol-test cases? Yeah, there's Rest-Json versions of [these test cases](https://github.com/smithy-lang/smithy-ruby/blob/702f6c5012e6dc439d7b9dcc4b59d1e3df306515/codegen/smithy-ruby-rails-codegen-test/model/protocol-test/http-headers.smithy#L30-L137)

Reply:
I think v4 also won't need an update for this change since restjson/restxml rely on this base class, so I think those test cases shoudl continue to pass.   (once we get v4 protocol tests passing, I'd like to add a "downstream" test to the CI so that we at least know when our smithy-ruby changes break v4 and require a change there).

Reply:
I was asking if the rest json builders will change. It should generate new code for that right?

Reply:
Yup, yes any changes we make here will update the rest-json builder